cmd/vic-machine/configure/configure.go

```diff
@@ -325,6 +325,8 @@ func (c *Configure) Run(clic *cli.Context) (err error) {
 		log.Errorf("Configuring cannot continue - failed to create validator: %s", err)
 		return errors.New("configure failed")
 	}
+	defer validator.Session.Logout(ctx)
```

Comment:
there is one case that session is created successfully, but still returns error in the NewValidator method. Might also add one defer in that method